src/accelerate/accelerator.py

```diff
@@ -2727,6 +2728,7 @@ def save_model(
         >>> accelerator.save_model(model, save_directory)
         ```
         """
+        from huggingface_hub import split_torch_state_dict_into_shards
```

Comment:
hf_hub is a required dep of accelerate, I think we can move this import up

Reply:
Done !